server/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverRequest.java

```diff
@@ -290,6 +388,20 @@ public String getNewIndexName() {
         return newIndexName;
     }
 
+    public boolean areConditionsMet(Map<String, Boolean> conditionResults) {
```

Comment:
Can you add javadocs for this please?

Reply:
+1, 4d1f3c10beff19d4bbe6f3344ed5cfa812e591a3